lisa/sut_orchestrator/azure/features.py

```diff
@@ -515,25 +516,25 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self._initialize_information(self._node)
         self._is_nvidia = True
 
-    @classmethod
-    def _install_by_platform(cls, *args: Any, **kwargs: Any) -> None:
-        template: Any = kwargs.get("template")
-        environment = cast(Environment, kwargs.get("environment"))
-        log = cast(Logger, kwargs.get("log"))
-        log.debug("updating arm template to support GPU extension.")
-        resources = template["resources"]
-
-        # load a copy to avoid side effect.
-        gpu_template = json.loads(cls._gpu_extension_template)
-
-        node: Node = environment.nodes[0]
-        runbook = node.capability.get_extended_runbook(AzureNodeSchema)
-        if re.match(cls._amd_supported_skus, runbook.vm_size):
-            # skip AMD, because no AMD GPU Linux extension.
-            ...
+    def _install_driver_using_platform_feature(self) -> None:
```

Comment:
_install_driver_using_extension?

Reply:
I wanted to keep the name generic because I was not sure if other platforms (like AWS) calls the similar feature as extension.
But "_install_driver_using_platform_feature" is a very confusing name.